gen/eduroam_block.pl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+
+local $::SERVICE_NAME = "eduroam_block";
```

Comment:
Please drop file extension ``.pl`` since perun-engine component expects scripts without it.

Also please make script executable by command in a console: ``chmod +x gen/eduroam_block.pl``